schema/network_attach.yaml

```diff
@@ -65,6 +65,15 @@ spec:
               ...
             }
           ]
-
   ...
-
+  spec:
+    containers:
+    - name:
+    ...
+      resources:
+      # In case of sriov interface type, resources must be requested from the Intel SR-IOV Netwrok Device plugin 
```

Comment:
same. any devices related to network management can be requested via this API, not just SR-IOV VFs